src/FormBuilder/DataTransferObjects/EmailNotificationData.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Give\FormBuilder\DataTransferObjects;
+
+use Give\FormBuilder\Actions\ConvertNotificationToDTO;
+
+/**
+ * @since 0.1.0
+ */
+class EmailNotificationData
+{
+    /** @var string */
+    public $id;
+
+    /** @var string */
+    public $title;
+
+    /** @var array */
+    public $statusOptions;
+
+    /** @var bool */
+    public $supportsRecipients;
+
+    /** @var array */
+    public $defaultValues;
+
+    public static function fromLegacyNotification($notification): EmailNotificationData
+    {
+        return (new ConvertNotificationToDTO($notification))->toDto();
```

Comment:
If this is specific to legacy we should rename to `ConvertLegacyNotificationToEmailNotificationData`.

Also in light of being an action this should not need to call `toDto()` and instead by invoking should return the DTO.

Reply:
Works for me - will do.